recipes/lpython/bld.bat

```diff
@@ -0,0 +1,17 @@
+call build0.bat
+if errorlevel 1 exit 1
+
+cmake ^
+    -G Ninja ^
+    -DCMAKE_BUILD_TYPE=Release ^
+    -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+    -DXJUPYTER_DATA_DIR=%PREFIX%\\share\\jupyter ^
```

Comment:
Shouldn't this be `LIBRARY_PREFIX`? Not sure

Reply:
I can debug it once this is in. It seems to work with LFortran, so I used the same here.